source/testing/testing-models.md

```diff
@@ -50,7 +50,7 @@ module('Unit | Model | player', function(hooks) {
 ```
 
 Also note, how both creating a record and updating properties on the record through the `levelUp` method requires
-us to wrap these operations into a `run` function. You can read more the Ember run loop [over here](https://guides.emberjs.com/v2.18.0/applications/run-loop/).
+us to wrap these operations into a `run` function. You can read more the Ember run loop [over here](https://guides.emberjs.com/v3.0.0/applications/run-loop/).
```

Comment:
This could probably be https://guides.emberjs.com/current/applications/run-loop/